app/components/Welcome.js

```diff
@@ -19,25 +32,56 @@ export default class Welcome extends Component {
     setNotFirstTimeFlag()
   }
 
+  selectedLanguage (i) {
+    // set the language to the selected value
+    i18n.changeLanguage(this.state.options[i].code)
+    setDefaultLanguage(this.state.options[i].code)
+  }
+
+  onLanguageChange (event) {
+    // Gets the index of the selected option
+    this.selectedLanguage(event.target.selectedIndex)
+  }
+  renderOptions () {
+    let options = [...SUPPORTED_LANGS]
+    this.state.options = options
+
+    return options.map((lang, index) => (
+      <option key={index} index={index} value={lang.code}>
+        {lang.language}
+      </option>
+    ))
+  }
+
   render () {
     // Index value has not been set, so there is nothing to show.
     if (this.props.index === -1) {
-      return (<Spinner />)
+      return <Spinner />
```

Comment:
Just keep them or change all returns in render in the whole project to keep it consistent

Reply:
I don't understand what you are referring to, i did't touch the spinner, but please explain me.

Reply:
He is referring to the parenthesis that were removed around the Spinner component :slightly_smiling_face: